.circleci/config.yml

```diff
@@ -157,17 +157,25 @@ jobs:
             public-read --recursive --exclude "*" --include "*.zip"
   deploy_docs:
     docker:
-      - image: 'redislabsmodules/rmbuilder:latest'
+      - image: 'python:buster'
     steps:
+      - run:
+          name: Setup executor
+          command: |
+            apt-get -qq update
+            apt-get -q install -y git openssh-client curl ca-certificates make tar gzip wget
+            cd ..; curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"; unzip awscliv2.zip;./aws/install
```

Comment:
Looking at the commands executed below, I don't understand why do we install:
`git openssh-client curl ca-certificates make tar gzip wget`

Reply:
Because it works? Is that a blocker?

I think that we can loose make, tar and wget. The rest maybe should stay.

Reply:
/cc @rafie

Reply:
ca-certificates is needed for curl/wget.
openssh-client is needed for the CircleCI executor to work (when one starts from a non-CircleCI standard image, things do not work out of the box, so packages needs to be added).
The bottom line is that it's a standard way to configure a non-standard CircleCI executor. I probably can encapsulate that in a script, so feel free to rub out make/wget/tar if you don't need them.

Reply:
Removing `make`, `wget` and `tar`
Thanks!